tests/unit_tests/tests_launch/test_job.py

```diff
@@ -0,0 +1,21 @@
+from wandb.apis.public import Api as PublicApi
+
+
+def test_job_call(user, wandb_init, test_settings):
```

Comment:
I think this test is really close, but doesn't quite actually 100% test what we set out to. Does it prove that we haven't messed up the call signature? yes. However, we don't actually check that the queue_name passed is where the queued run ended up! We could so something more explicit like in this test: https://github.com/wandb/wandb/blob/c783fa85914f4da82135b28c5ec0e4fe922cf9ab/tests/unit_tests/tests_launch/test_launch_add.py#L219

Reply:
Updated to pop from the run queue! Let me know if anything else could be improved.